.github/workflows/linux_ci.yml

```diff
@@ -33,7 +33,10 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install onnx onnxruntime requests
+          # TODO: now ONNX only supports Protobuf <= 3.20.1
+          python -m pip install protobuf==3.20.1
+          python -m pip install onnx onnxruntime requests py-cpuinfo
+          python -m cpuinfo
```

Comment:
Why run cpuinfo here?
If you want to keep it, could you please add a comment?

Reply:
I kept it intentionally for future debugging since the CPU information can be very helpful while debugging why ORT inference behaves differently on different machines. I will send a PR to add comments about it.